### PR TITLE
Add doc about version usage

### PR DESCRIPTION
**Pull request by mosabua**

### CODE REVIEW

**Comment by mosabua** [comment]
This is mostly for myself so I dont forget when publishing and can look it up easily without digging into the scripts. Please merge @electrum

File: presto-docs/README.md [hunk 1]
```diff
@@ -114,6 +122,23 @@ make clean livehtml
...
+
+The version displayed in the resulting HTML is read from the top level Maven
+`pom.xml` file `version` field, by default.
+
+To deploy a specific documentation set (e.g. a SNAPSHOT version) as release
+version you have to override the pom version with the `PRESTO_VERSION`
+environment variable.
+
+```bash
+export PRESTO_VERSION
```
Comment (electrum) [inline presto-docs/README.md]: I think it would be better to write this inline with a version specified:
```
PRESTO_VERSION=1234 make clean html

Comment (electrum) [reply presto-docs/README.md]: @mosabua I'll merge this now, but please send a PR for this if you agree

Comment (mosabua) [reply presto-docs/README.md]: https://github.com/prestosql/presto/pull/2244

**Pull request merged**